core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -98,8 +107,8 @@ class Validator private (private val _name : String, val startStep : Step, val c
 
 
   private val platformMBeanServer = ManagementFactory.getPlatformMBeanServer()
-  private val objectName = new ObjectName("\"com.rackspace.com.papi.components.checker\":type=\"Validator\",scope=\""+
-                                 name+"\",name=\"checker\"")
+  private val objectName = new ObjectName("com.rackspace.com.papi.components.checker:type=Validator,scope="+
+                                 name+",name=checker")
```

Comment:
Got it. Are quotes valid characters, though? Or, I guses I should ask, what are these strings used for? If it's just for internal stuff, whatever. If it's user-facing, then a backwards incompatible change makes me a little nervous.